src/Bladeburner/Actions/Casualties.ts

```diff
@@ -0,0 +1,51 @@
+import { getRandomIntInclusive } from "../../utils/helpers/getRandomIntInclusive";
+import { OperationTeam } from "../Bladeburner";
+
+export type Casualties = {
+  deaths: number; // Actual roll
+  damagedSleeves: number; // Sleeves included in actual deaths
+};
+
+type CasualtyRolls = {
```

Comment:
I don't see any actual uses of this type? If it's just for testing, it probably shouldn't be added - you should think of a different way to get effective tests.

Reply:
The kill doesn't need the rolls, I prefer this over ducktyping.

Reply:
The intention is to separate the types from "what could happen, ie. potential" from "what did happen"

I can separate the types, but the correct hinting results in more code being added on the Bladeburner side. Though now that I'm looking at it from a fresh perspective, I may just move this to a method and hint the type of the parameter (similar to how Bladeburner is being passed to Action.ts getters, though a step I disagree with design-wise)